framework/console/controllers/QueueController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\console\controllers;
+
+use Yii;
+use yii\console\Controller;
+
+/**
+ * Runs the queue listener.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+class QueueController extends Controller
```

Comment:
@klimov-paul Why you think so? To be honest its main class with 90% of functionality. Example:

```
./yii queue/listen
./yii queue/listen --queue=emails
./yii queue/listen --queue=avatars
```